soap/src/java/com/zimbra/soap/mail/message/RestoreContactsResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2017 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.soap.mail.message;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.zimbra.common.soap.MailConstants;
+
+@XmlAccessorType(XmlAccessType.NONE)
+@XmlRootElement(name = MailConstants.E_RESTORE_CONTACTS_RESPONSE)
+public class RestoreContactsResponse {
+
+    /**
+     * @zm-api-field-tag status
+     * @zm-api-field-description Restore request status
+     */
+    @XmlAttribute(name = MailConstants.A_STATUS, required = true)
+    private String status;
```

Comment:
Could this be an enum?

Reply:
Done

Reply:
Yes, agreed. Changed the response attribute itself to enum. Here is the PR: https://github.com/Zimbra/zm-mailbox/pull/355